app/serializers/component_serializer/paragraph_component_serializer.rb

```diff
@@ -23,15 +21,7 @@ def name
     end
 
     def data
-      @content.map do |content|
-        {}.tap do |hash|
-          hash[:content] = content[:content]
-
-          data = content.try(:except, :content)
-
-          hash[:data] = data if data.present?
-        end
-      end
+      @content if @content
```

Comment:
What if you provide no content, will the serialiser still work with blank content?

Reply:
You are right, It should not work if it is not given content. That would be the minimum requirement. I will take away that conditional.